eahub/profiles/migrations/0004_auto_20190312_1918.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2b1 on 2019-03-12 19:18
```

Comment:
Please rename this file to something more informative. `0004_profile_other_fields.py` would be good if you don't happen to have any better ideas.

Reply:
done